### ISSUE TITLE
Add rendering for leisure=bleachers

### ISSUE THREAD

**Issue by Tomasz-W** [gravitystorm/openstreetmap-carto#3140]
Quite useful feature, it would inform map users about possibility to sit down on a tribune.
https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dbleachers

Currently, this tag is not rendered, so  people are sometimes tagging outdoor tribunes as building=grandstand, which is a mistake (Tagging for the renderer)

To make it visible and recognizable, the color should be different than leisure=stadium/ sport_centre and leisure=pitch/ track::
https://wiki.openstreetmap.org/w/images/8/8e/Area_amenity%3Dschool.png
https://wiki.openstreetmap.org/w/images/f/fd/Area_leisure%3Dtrack.png

Examples of these objects:
http://www.nagawki.pl/gallery/Amfiteatr.jpg
https://d-pt.ppstatic.pl/kadry/k/r/1/17/46/55b7a17d0c389_o,size,1068x623,q,71,h,181ae4.jpg
https://d-nm.ppstatic.pl/kadr/k/r/fd/dd/57bf29eda3e82_o,size,933x0,q,70,h,a2d028.jpg

**Comment by SomeoneElseOSM** [comment]
As noted at the wiki page https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dbleachers , "bleachers" isn't an English word but an American one.  The nearest equivalent English word is "grandstand" or "stands".  See this tagging list discussion: https://lists.openstreetmap.org/pipermail/tagging/2017-October/thread.html#33954 .  Also the use of the word "tribune" to mean "a raised area to stand" isn't in common use in English (I've never heard it used in that context at all, although a copy of Chambers on the shelf does have it).

**Comment by Tomasz-W** [comment]
@SomeoneElseOSM Did you mean that `building=grandstand` is equal to `leisure=bleachers`? 
When you Google these both words, you will receive a little bit different results:
* grandstand almost always have a roof above:   https://www.google.pl/search?client=opera&hs=YES&biw=1920&bih=981&tbm=isch&sa=1&ei=97yyWvneF-KF6AS52IWQAQ&q=grandstand&oq=grandstand&gs_l=psy-ab.3..0i67k1j0j0i7i30k1l8.795.795.0.1007.1.1.0.0.0.0.105.105.0j1.1.0....0...1c.1.64.psy-ab..0.1.105....0.QS0cUyhIEQU
* while bleachers almost always doesn't have it: https://www.google.pl/search?q=bleachers&client=opera&hs=xtm&source=lnms&tbm=isch&sa=X&ved=0ahUKEwjp8Y6inf7ZAhVKyqYKHampBqgQ_AUICigB&biw=1920&bih=981

Problem to resolve is to clear the situation, and have 2 tags: one for big tribunes where you can enter inside (because of it's levels or shops etc. on the ground level) and second one for smaler ones, which are outdoor landcover.
I'm not stick to opinion that `leisure=bleachers` is proper tag for outdoor tribunes or seating steps, so I'm open to discussion.

**Comment by SomeoneElseOSM** [comment]
> Did you mean that building=grandstand is equal to leisure=bleachers?

No, I meant that "bleachers" isn't used in English English at all, except as an American import to describe (say) the seating at a baseball ground in the USA.  Whether a seating has a roof or not doesn't really mean it does or doesn't get called a "grandstand" or stop you being "in the stand".  Winter game venue stands (and newer ones) are more likely to have roofs that summer game ones.  Often you might get only a partial roof.  Here's Lord's (cricket) for example: https://www.flickr.com/photos/james_mans/1469161659 , and here's Le Mans https://lemansrace.com/le-mans-24-hours-grandstand-seats/ , which are both like that.

**Comment by matkoniecz** [comment]
Given low usage (less than 2000 worldwide) and issues mentioned bt @SomeoneElseOSM I propose to decline it.

If somebody is interested in mapping this feature I would strongly encourage to discuss fixing issue mentioned by @SomeoneElseOSM

**Comment by jeisenbe** [comment]
Reopening, usage has increased sharply to over 6000 features, and there seems to be a clear distinction between this and building=grandstand according to the current wiki page description: https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dbleachers

**Comment by Tomasz-W** [comment]
I was trying some possible options in Photoshop before I opened this issue, and pitch colour with some gray grid overlay I find the best, mainly because it would fit good a pitches which bleachers ussually are placed next to.
![Untitled-1](https://user-images.githubusercontent.com/25656654/67955586-19ccfd00-fbf3-11e9-9e16-7c99f28cf38d.jpg)
![Untitled-2](https://user-images.githubusercontent.com/25656654/67955588-1a659380-fbf3-11e9-8c95-750b61df0831.jpg)

**Comment by jeisenbe** [comment]
Thank you, @Tomasz-W. 

While bleachers are often near pitches, they can also be found at outdoor theatres and other performance sites, and the feature itself is not similar to a pitch.

I would suggest that `leisure=bleachers` is most similar to buildings, bridges and pedestrian areas, considering the area features that are currently rendered: they are a structure which can be used by people, and are only accessible to people on foot. Ideally we might design a rendering that looked somewhat similar to buildings and highway=pedestrian areas (and highway=platform / railway=platform).

**Comment by CorruptComputer** [comment]
Would it be possible to get an update on the status of this? 
I recently mapped part of this location: https://www.openstreetmap.org/edit#map=19/34.12030/-118.06490
Without the these being rendered it looks a bit strange here to me, and I think having this extra context would be great.
